### ISSUE TITLE
通过实体类生成数据库sql的方法可以移植一下,做一个生成与数据库无关的Bean的方法吗?

### ISSUE THREAD

**Issue by hunterlee01123** [gejun123456/MyBatisCodeHelper-Pro#729]
![image](https://user-images.githubusercontent.com/47778938/146912739-4f2d6bff-9c92-437b-a141-c99c034c0faa.png)
![image](https://user-images.githubusercontent.com/47778938/146913178-bae12705-440b-4dc7-b04d-5769b5ef0057.png)
就是这个功能,好多bean属性很多,上上下下操作输入好麻烦,加注释敲/**也很烦.网上也没有搜索到和你的方法界面一样好的模板生成工具.有满足要求的类似工具也可以推荐下

**Comment by gejun123456** [comment]
是指快速给实体的属性添加注释这些么

**Comment by gejun123456** [comment]
给实体快速添加注解，注释吧

**Comment by hunterlee01123** [comment]
> 

是的,包括给字段标明数据类型,为字段命名啥的.   或者说我就是想在这个面板里把一个实体类初始化.注解只要有jsonformat相关的就好

**Comment by gejun123456** [comment]
嗯嗯，可以弄个单独的插件实现这些
